site/src/App/routes/examples/basic-form/basic-form.tsx

```diff
@@ -0,0 +1,425 @@
+            </Stack>
+          </Card>
+        </Step>
+
+        <Step
+          heading="6. Add the button"
+          detail={
+            <Text>
+              Now that we have all of our form fields in the card, we can now
+              add a <TextLink href="/components/Button">Button</TextLink> to
+              submit the form.
+            </Text>
+          }
+        >
+          <Card>
+            <Stack space="large">
+              <Heading level="3">Add role</Heading>
+              <Autosuggest
+                label="Job title"
+                suggestions={[
+                  { text: 'Developer' },
+                  { text: 'Designer' },
+                  { text: 'Product Manager' },
+                ]}
+              />
+              <TextField label="Company name" />
+              <MonthPicker label="Started" />
+              <Textarea
+                label="Description"
+                secondaryLabel="recommended"
+                description="Summarise your responsibilities, skills and achievements."
+              />
+              <Button>Save</Button>
+            </Stack>
+          </Card>
+        </Step>
+
+        <Step
+          detail={
+            <Stack space="large">
+              <Text component="p">
+                You&rsquo;ll notice the button is the full width of the card.
+                From a mobile perspective this is fine, but on a larger device
+                this is not ideal.
+              </Text>
+              <Text component="p">
+                We can address this by wrapping the button in an{' '}
+                <TextLink href="/components/Actions">Actions</TextLink>{' '}
+                component. Now it will be full width on mobile and only as wide
+                as it&rsquo;s content larger screens.
+              </Text>
+            </Stack>
+          }
+        >
+          <Card>
+            <Stack space="large">
+              <Heading level="3">Add role</Heading>
+              <Autosuggest
+                label="Job title"
+                suggestions={[
+                  { text: 'Developer' },
+                  { text: 'Designer' },
+                  { text: 'Product Manager' },
+                ]}
+              />
+              <TextField label="Company name" />
+              <MonthPicker label="Started" />
+              <Textarea
+                label="Description"
+                secondaryLabel="recommended"
+                description="Summarise your responsibilities, skills and achievements."
+              />
+              <Actions>
+                <Button>Save</Button>
+              </Actions>
+            </Stack>
+          </Card>
+        </Step>
+
+        <Step
+          heading="7. Polish!"
+          detail={
+            <Stack space="large">
+              <Text>
+                Now that we have all our elements in place we can polish until
+                we are happy. Adjusting white space between elements, or even
```

Comment:
How about `Adjusting white space between elements across mobile, tablet and desktop to achieve the desired goal.`?